pkg/controllers/node/hostendpoints.go

```diff
@@ -28,6 +28,44 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
+	defaultProfileName = "allow"
```

Comment:
We should pick something a little more collision-resistant, like `org-projectcalico-allow` or something.